api/src/main/java/javax/mvc/MvcContext.java

```diff
@@ -123,4 +125,12 @@
      * @return instance of encoders.
      */
     Encoders getEncoders();
+
+    /**
+     * Returns the locale of the current request.
+     *
+     * @return The request locale
+     */
```

Comment:
I think you need the first sentence with a dot (.) at the end to avoid the method having empty description in higher level docs (class level).